src/EllCrv/Misc.jl

```diff
@@ -219,6 +272,29 @@ function smod(a::T, b::S) where {T, S}
   return z
 end
 
+function normal_elem(K::T, L::T) where T<:FinField
```

Comment:
As we had to look it up: please provide a comment/ doc string to say what a normal_elem actually is

Reply:
I'll add it.